Kernel/run

```diff
@@ -38,6 +38,7 @@ else
         -device e1000,netdev=breh \
         -kernel kernel \
         -hda _fs_contents \
-        -soundhw pcspk
+        -soundhw pcspk \
+    	-debugcon stdio
```

Comment:
This change is unrelated and should not be in the PR.
Also, if you pull master, you'll find fba57d6ba3f0b5fc56d01874df2f37a0d21a32c3 that enabled this by default :)